build/LegacyPackages/14.0/Microsoft.Build.Framework/Microsoft.Build.Framework.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata minClientVersion="2.8.1">
```

Comment:
I'm going to remove the minClientVersion

---
In reply to: [75906453](https://github.com/Microsoft/msbuild/pull/929#discussion_r75906453) [](ancestors = 75906453)